notificationdemo/source/c/build.sh

```diff
@@ -15,35 +15,100 @@
 #  limitations under the License.
 #
 
-# Exits immediately if error occurs
+# Exit immediately if error occurs
 set -e
 
 RUN_DIR=`pwd`
 
-help() {
+help_message() {
     echo "Choose one of the following: {build|run|deploy|clean}"
+    echo "Supported targets: posix, cc32xx, esp8266"
     exit 1
 }
 
 if [ $# -eq 0 ]
 then
-    help
+    help_message
 fi
 
-APP_NAME="demo_client"
 PROJECT_HOME=$(pwd)
 BUILD_DIR="build"
+KAA_LIB_PATH="./libs/kaa"
+KAA_C_LIB_SRC_PATH="$KAA_LIB_PATH/src"
+KAA_SDK_TAR="kaa-c*.tar.gz"
 KAA_TOOLCHAIN_PATH_SDK=""
+# TODO: comment about inconvenience between KAA_TARGET and KAA_PLATFORM
```

Comment:
I propose to remove this TODO since the `build.sh` scripts will be eliminated ASAP